api/default_networks.go

```diff
@@ -18,19 +17,39 @@ const (
 	ArbitrumSepoliaChainID uint64 = 421614
 	sntSymbol                     = "SNT"
 	sttSymbol                     = "STT"
+	proxyNodefleet                = "proxy-nodefleet"
```

Comment:
nit: declare alias type to group all "provider IDs" or add some indication in the const name that we're talking about provider IDs